torch/csrc/api/include/torch/data/dataloader_options.h

```diff
@@ -62,14 +65,17 @@ struct FullDataLoaderOptions {
         max_jobs(options.max_jobs_.value_or(2 * workers)),
         timeout(options.timeout_),
         enforce_ordering(options.enforce_ordering_),
-        drop_last(options.drop_last_) {}
+        drop_last(options.drop_last_),
+        chunk_loading(options.chunk_loading_) 
+        {}
 
   size_t batch_size;
   size_t workers;
   size_t max_jobs;
   optional<std::chrono::milliseconds> timeout;
   bool enforce_ordering;
   bool drop_last;
+  bool chunk_loading;
```

Comment:
This is pre-existing but I would like to get this straight for future development: some class member variables are suffixed with "_" but some are not. What is the guideline on naming convention for PyTorch?

Reply:
Private fields get trailing underscores, public fields don't